doc/user-guide/user-guide.rst

```diff
@@ -211,9 +210,8 @@ This image uses the ``deploy_lsstsqre`` as a base image and uses ``eups`` to ins
 
 Because both ``Scheduler`` and ``MTAOS`` depends on ``lsst_sims`` they are built from a different base image than the base images above.
 We then proceed to build the ``lsst_sims`` image, which will install ``lsst_sims`` on ``deploy_lsstsqre``.
-Once this step is done, it is also possible to build ``aos_aoclc`` which is another intermediate stage in building ``MTAOS``.
 
-With ``lsst_sims`` and ``aos_aoclc`` built, it is then possible to build ``scheduler`` and ``mtaos`` in any order.
+With ``lsst_sims`` built, it is then possible to build ``scheduler`` and ``mtaos`` in any order.
```

Comment:
We don't need sims to run mtaos anymore. So mtaos can be build as soon as `deploy_lsstsqre` is done.

Reply:
Done